lib/backend/k8s/conversion/conversion.go

```diff
@@ -171,13 +162,6 @@ func (c Converter) PodToWorkloadEndpoint(pod *kapiv1.Pod) (*model.KVPair, error)
 	// the host-side veth configured by the CNI plugin.
 	interfaceName := VethNameForWorkload(wepName)
 
-	// Build the labels map.
-	labels := map[string]string{}
-	if pod.ObjectMeta.Labels != nil {
-		labels = pod.ObjectMeta.Labels
-	}
-	labels["calico/k8s_ns"] = pod.ObjectMeta.Namespace
```

Comment:
I think we still need to set the "magic" labels in this function - there are other places in our code - probably just kube-controllers - that relies on this function returning the same set of labels that you would see when doing a List/Get through libcalico.

Reply:
sounds good.

in that case - i'll just make sure we set the labels in the clientv2 as well and persist them and then remove the syncer magic.